README.md

```diff
@@ -40,19 +40,19 @@ A curated list of awesome research papers, datasets and software projects devote
 
 ## Digests
 
-* [Learning from "Big Code"](http://learnbigcode.github.io) - Techniques, challenges, tools, datasets on "Big Code".
+* [Learning from "Big Code"](https://learnbigcode.github.io) - Techniques, challenges, tools, datasets on "Big Code".
```

Comment:
at least my browser render is broken at `https://learnbigcode.github.io`. @m09 , @vmarkovtsev - do yo have this issue?

Reply:
yes mine is also broken

Reply:
yes mine is also broken

Reply:
yes I also have this issue, good catch!

Reply:
Can confirm, `https://` version is not rendered prpoperly

Reply:
Can confirm, `https://` version is not rendered properly

Reply:
Can confirm, `https://` version is not rendered properly

Reply:
Can confirm, `https://` version is not rendered properly

Reply:
Can confirm, `https://` version is not rendered properly